### PR TITLE
Sk 60 mod support

### PR DESCRIPTION
**Pull request by Turbolious**
https://github.com/BAAS-Dynamic/DCS-SK60-Mod/releases

### CODE REVIEW

File: resources/units/aircraft/SK-60.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+carrier_capable: false
+description:
+  The Saab 105 is a Swedish high-wing, twinjet trainer aircraft developed in the early 1960s 
+  as a private venture by Saab AB.[2] The Swedish Air Force, which had opted to procure the type 
+  for various roles, issued the aircraft with the designation Sk 60. The Sk 60 entered service in 1967, 
+  replacing the ageing De Havilland Vampire fleet.
+  SK-60B is a two-seat attack version for the Swedish Air Force, modified from Sk 60A with new weapons sight
+  
+introduced: 1968
```
Comment (Raffson) [inline resources/units/aircraft/SK-60.yaml:10]: Interesting, the description states it entered service in 1967, but you're actually taking 1968 😅 
Any particular reason for this?

File: resources/factions/sweden_1980.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+  "country": "Sweden",
+  "name": "Sweden 1980",
+  "authors": "Turbolious",
+  "description": "<p>Sweden in the 80s.</p><p>Note : Limited A2A capabilities.</p>",
+  "locales": [
+    "sv_SE"
+  ],
+  "aircrafts": [
```
Comment (Raffson) [inline resources/factions/sweden_1980.json:9]: I'm missing a Draken in this section 😝

File: resources/units/aircraft/SK-60.yaml
Comment (Raffson) [inline resources/units/aircraft/SK-60.yaml:1]: Missing radio config since it's a flyable aircraft 😅

**Review by Raffson — CHANGES_REQUESTED** [review]
Also missing a changelog entry 😅

File: resources/units/aircraft/SK-60.yaml (hunk 1 above)
Comment (Turbolious) [reply resources/units/aircraft/SK-60.yaml:10]: Yea, SK-60 entered service in 67 and the SK-60B light attack came out the next year, which is the one we are using.

File: resources/factions/sweden_1980.json (hunk 2 above)
Comment (Turbolious) [reply resources/factions/sweden_1980.json:9]: If i keep learning new stuff, who knows. I do know i can just pull it in from War thunder and make a mod out of it. I would just have to research how they make the FC3 mods lolol

File: resources/units/aircraft/SK-60.yaml
Comment (Turbolious) [reply resources/units/aircraft/SK-60.yaml:1]: ahh yes. will add in one moment....or 2 weeks

File: resources/units/aircraft/SK-60.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+role: COIN / FAC /Light Attack
+gunfighter: true
+variants:
+  SK-60: {}
+radios:
+  intra_flight: AN/ARC-164
+  inter_flight: R&S M3AR UHF
+  channels:
+    type: common
+    namer: viper
```
Comment (Raffson) [inline resources/units/aircraft/SK-60.yaml]: I believe you can drop this line

File: resources/units/aircraft/SK-60.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+manufacturer: Saab AB
+origin: Sweden
+price: 4
+role: COIN / FAC /Light Attack
+gunfighter: true
+variants:
+  SK-60: {}
+radios:
+  intra_flight: AN/ARC-164
+  inter_flight: R&S M3AR UHF
```
Comment (Raffson) [inline resources/units/aircraft/SK-60.yaml]: Do we have support for this radio? Otherwise we'll need to add it to radios.py 😅

**Review by Raffson — CHANGES_REQUESTED** [review]

**Comment by Raffson** [comment]
After having discussed the radios, it turns out there's no radio-panel available for the SK-60, thus the radio configuration is to be dropped 👍

**Review by Raffson — APPROVED** [review]

**Pull request merged**